rebaser_test.go

```diff
@@ -54,7 +55,8 @@ func testRebaser(t *testing.T, when spec.G, it spec.S) {
 		additionalNames = []string{"some-repo/app-image:foo", "some-repo/app-image:bar"}
 
 		rebaser = &lifecycle.Rebaser{
-			Logger: &log.Logger{Handler: &discard.Handler{}},
+			Logger:      &log.Logger{Handler: &discard.Handler{}},
+			PlatformAPI: api.MustParse("0.4"),
```

Comment:
Could we use something like `api.Platform.Latest()` here? The tests that require the API version to be lower than some boundary could then set it explicitly, otherwise it would just be latest. This is a pattern that we started here: https://github.com/buildpacks/lifecycle/blob/d9c7de5395c1ba66682b3e548f8955a11b265c17/builder_test.go#L67 and should be helpful as we support ever more api versions.

Reply:
Good idea.
I'll make this change both here and in `exporter_test.go`.

Reply:
Please see https://github.com/buildpacks/lifecycle/pull/498/commits/721cf13f78654cd7978864642557e02ce6a9931d